lib/airbrake/rails/active_record_subscriber.rb

```diff
@@ -0,0 +1,26 @@
+module Airbrake
+  module Rails
+    # ActiveRecordSubscriber sends SQL information, including performance data.
+    #
+    # @since v8.1.0
+    class ActiveRecordSubscriber
+      def initialize(notifier)
+        @notifier = notifier
+      end
+
+      def call(*args)
+        return unless (route = Thread.current[:airbrake_rails_route])
+
+        event = ActiveSupport::Notifications::Event.new(*args)
+        @notifier.notify_sql(
+          route: route,
+          method: Thread.current[:airbrake_rails_method],
+          sql: event.payload[:sql],
```

Comment:
Is this query normalized? e.g. `SELECT * FROM some WHERE id = ?` instead of `SELECT * FROM some WHERE id = 123`

Reply:
Not normalised. Do we want to normalise it here or on the backend?

Reply:
If Rails provides normalized query then we should use it and send to the backend normalized query. If not we probably have to normalize queries on the server...